core/txn/command.go

```diff
@@ -106,6 +106,15 @@ func (cmd Command) IsValidatorCommand() bool {
 	}
 }
 
+func (cmd Command) IsTTLCommand() bool {
+	switch cmd {
+	case SubmitOrderCommand, CancelOrderCommand, AmendOrderCommand, WithdrawCommand, LiquidityProvisionCommand, CancelLiquidityProvisionCommand, AmendLiquidityProvisionCommand, DelegateCommand, UndelegateCommand, StateVariableProposal, ProposeCommand, VoteCommand, TransferFundsCommand, CancelTransferFundsCommand:
```

Comment:
just curious why is it not true for all except for validator commands?

Reply:
Things like the `AnnounceNodeCommand` aren't validator commands, and I'm not sure if we should, essentially `&>/dev/null` those commands after a number of blocks

Reply:
After talking it through some more, we're just going to use `!IsValidatorCommand()` instead